examples/mqtt/mqtt_shared_feeds.py

```diff
@@ -70,5 +70,5 @@ def message(client, feed_id, payload):
 while True:
     value = random.randint(0, 100)
     print('Publishing {0} to {1}.'.format(value, IO_FEED))
-    client.publish(IO_FEED, value, IO_FEED_USERNAME)
+    client.publish(IO_FEED, value, feed_user=IO_FEED_USERNAME)
```

Comment:
Do we need the `feed_user` kwarg here?

Reply:
Appears to be necessary. I had issues unrelated (unauthorized shared feed) when testing this, but when inspecting the method it's third argument is group_name and fourth is feed_user (in theory for shared feeds).

Reply:
Okay!